srcpkgs/amule/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'amule'
+pkgname=amule
+version=11066
+revision=1
+wrksrc=amule-master
+build_style=gnu-configure
+configure_args="--prefix=/usr --mandir=/usr/share/man
```

Comment:
`--prefix` and `--mandir` are set by default for the `gnu-configure` build-style.

Please indent inside of variables with just one space instead of tab.

Reply:
Fixed.

Reply:
Fixed